src/writer_eventbridge.py

```
@@ -2,41 +2,41 @@
+                    "Detail": json.dumps(message),
+                    "EventBusName": EVENT_BUS_ARN,
                 }
             ]
         )
         if response["FailedEntryCount"] > 0:
-            msg = str(response)
-            _logger.error(msg)
-            return False, msg
+            _logger.error(str(response))
+            return False
     except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)